client/params/config_test.go

```diff
@@ -23,7 +23,7 @@ func TestCheckCompatible(t *testing.T) {
 			new:    MainnetChainConfig,
 			wantErr: &ConfigCompatError{
 				What:         "Chain ID",
-				StoredConfig: big.NewInt(1337),
```

Comment:
I'm not totally sure what this is, but all the tests still pass when it's changed.